### ISSUE TITLE
docs: EXpired Slack invite link

### ISSUE THREAD

**Issue by FavourDaniel** [apache/apisix#8907]
### Current State

In the documentation, the slack link is an expired one. and so new contributors cannot join the community.

### Desired State

It should be updated with one that works

**Comment by Amishakumari544** [comment]
Can you please show where it's expired? because in the `readme.md` it's working fine.

**Comment by Neilblaze** [comment]
> Can you please show where it's expired? because in the `readme.md` it's working fine.

@Amishakumari544 Yep, the link's working fine, but the SSO only authorizes emails registered under `apache.org` & I think she's trying Google's OAuth which is apparently failing.

**Comment by Amishakumari544** [comment]
@Neilblaze I joined through google login

**Comment by shreemaan-abhishek** [comment]
The link works perfectly fine for me, maybe because I have already logged in to apache's slack.

**Comment by Neilblaze** [comment]
@Amishakumari544 thanks for the highlight. Afaik, I also joined ASF via Google OAuth last year. But now, I think I found the root (thanks to @shreemaan-abhishek). I've got myself logged in on my _primary system_, and & I tried to *sign-in* on my _secondary system_ through the same email. But here's the catch. If someone's already a slack member for ASF, the SSO returns a 5xx validation err. 👇🏼 

![image](https://user-images.githubusercontent.com/48355572/220819339-e1516670-1b87-44bf-ac5f-7dfac57f1d65.png)

> In a 🥜, if you're already a member, signing up twice doesn't auto-redirect you to the channel.
